governance/coordination-structure.md

```diff
@@ -26,107 +23,136 @@ Toronto Mesh uses voting to determine whether we have reached consensus:
 
 ## Working Groups
 
+### Responsibilities of Working Group Leads
+
+Central Org
+  - Organize meetups and planning sessions
+  - Post event details to the tomesh.net website 
+
+Tools
+  - Maintain the tomesh.net website server 
+  - Maintain the Matrix server
+
+Deployment
+  - Coordinate node deployment efforts
+
+Website
+  - Maintain the tomesh.net website
+    - Update whenever security issues are found
```

Comment:
Probably don't need to list this, it's pretty minor, and even if we don't update security patches it's a static site and pretty safe